File: web-api/src/models/remote_connector/serial_remote_connector_parities_model.py
from enum import Enum


class  SerialRemoteConnectorParitiesModel(Enum):
    NONE = 'N'   # No parity
    EVEN = 'E'   # Even parity
    ODD = 'O'    # Odd parity
    MARK = 'M'   # Parity bit always 1
    SPACE = 'S'  # Parity bit always 0

File: web-ui/src/models/remote-connectors-settings/serial-remote-connector-parities-model.ts
/* eslint no-unused-vars: OFF */

export enum SerialRemoteConnectorParitiesModel {
    NONE = 'N',
    EVEN = 'E',
    ODD = 'O',
    MARK = 'M',
    SPACE = 'S'
}

export const SerialRemoteConnectorParities = [
    { id: SerialRemoteConnectorParitiesModel.NONE, description: 'Нет' },
    { id: SerialRemoteConnectorParitiesModel.EVEN, description: 'Четный' },
    { id: SerialRemoteConnectorParitiesModel.ODD, description: 'Нечетный' },
    { id: SerialRemoteConnectorParitiesModel.MARK, description: 'Всегда 1' },
    { id: SerialRemoteConnectorParitiesModel.SPACE, description: 'Всегда 0' },
]
